src/peergos/shared/user/TofuKeyStore.java

```diff
@@ -48,6 +48,13 @@ public TofuKeyStore() {
         return chains.getOrDefault(username, expired.getOrDefault(username, Collections.emptyList()));
     }
 
+    /**
```

Comment:
update the javadocs

Reply:
What do you mean? The critical thing here is the return value meaning.